doc/getting_started/getting_started.rst

```diff
@@ -20,22 +20,60 @@ Follow one of the following guides for your host operating system.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
-Clone the Zephyr Repository
-***************************
+.. _get_the_code:
+
+Get the source code
+*******************
+
+Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
+source code, instead of manually cloning the Zephyr repos along with West
+itself.
+
+Bootstrap West
+==============
+
+Install the bootstrapper for Zephyr's command-line tool, :ref:`west` in a
+shell or ``cmd.exe`` prompt:
+
+.. code-block:: console
+
+   # Linux
+   pip3 install --user west
 
-To clone the Zephyr source code repository from GitHub:
+   # macOS and Windows
+   pip3 install west
+
+.. note::
+   See :ref:`gs_python_deps` for additional clarfication on using the
+   `--user` switch.
+
+Additional information about West's structure can be found
+:ref:`here <west-struct>`.
+
+Clone the Zephyr Repositories
+=============================
+
+Clone the Zephyr source code repositories from GitHub using the ``west`` tool:
 
 .. code-block:: console
 
-   git clone https://github.com/zephyrproject-rtos/zephyr
+   mkdir zephyrproject
+   cd zephyrproject
+   west init .
```

Comment:
the directory defaults to cwd, so `west init` will suffice

Reply:
Yes I know, but I thought this would highlight the fact that one can specify the installation folder. I originally wrote `west init zephyrproject` but changed it to what it is now in order to avoid confusing people, who could perhaps not see at first sight that `zephyrproject` is just a folder name and not a special name you pass to `west` in order to clone Zephyr and not some other project.

Reply:
I think people will see this and assume the directory is mandatory, and I'm eager to minimize the number of "requirements" (perceived or real) needed to install zephyr.

Reply:
In that case, and if you want to minimize the number of steps too, should we have:
```
west init zephyrproject
```
instead?